### PR TITLE
feat: inputBox for minting

### PR DESCRIPTION
**Pull request by CL-Chen**

### CODE REVIEW

File: packages/app-vault/src/components/CollateralFaucetContainer/CollateralFaucet.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+export const ReadyCollateralFaucetStory: React.FunctionComponent = () => {
+  return (
+    <CollateralFaucet
+      faucetState={{
+        state: "READY",
+        account: "0x12345",
+        balance: BigNumber.from(0),
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        mintToken: () => {},
```
Comment (yehjxraymond) [inline packages/app-vault/src/components/CollateralFaucetContainer/CollateralFaucet.stories.tsx:22]: try using actions?

File: packages/app-vault/src/components/CollateralFaucetContainer/CollateralFaucetContainer.tsx [hunk 2]
```diff
@@ -1,32 +1,53 @@
...
 import { bnToNum } from "../../utils/number";
-export const CollateralFaucetContainer: FunctionComponent = () => {
-  const faucetState = useCollateralFaucet();
+
+export const CollateralFaucet: FunctionComponent<{ faucetState: VaultState }> = ({ faucetState }) => {
+  // added mintAmt and mintAmtChangeHanlder
+  const [mintAmt, setMintAmt] = useState(1000);
+
+  const mintAmtChangehandler = (event: React.ChangeEvent<HTMLInputElement>): void => {
+    setMintAmt(parseFloat(event.target.value));
```
Comment (yehjxraymond) [inline packages/app-vault/src/components/CollateralFaucetContainer/CollateralFaucetContainer.tsx:14]: have you tested it with non-number, negative number?

**Review by yehjxraymond — CHANGES_REQUESTED** [review]
Some comment

File: packages/app-vault/src/components/CollateralFaucetContainer/CollateralFaucet.stories.tsx (hunk 1 above)
Comment (CL-Chen) [reply packages/app-vault/src/components/CollateralFaucetContainer/CollateralFaucet.stories.tsx]: done

**Review by yehjxraymond — APPROVED** [review]
lgtm

**Pull request merged**